### DISCUSSION TITLE
Clickhouse vs Zookeeper

### DISCUSSION THREAD

**Discussion by Foxrider721** [ClickHouse/ClickHouse#37746] [Q&A]
Hello guys, 

I was wondering, is there significant difference between Zookeper and clickhouse keeper (From the docs I have seen RAFT algorithm is used by clickhouse keeper and  allows linearizability for reads and writes) is it the major advantages over zookeeper ? 
Is it more stable clickhouse keeper ? it is better suited for High I/O ?  
I'm planning to deploy clickhouse on k8s

**Comment by den-crane** [comment]
the major advantage is that keeper does not have "XID overflow" problem and use less RAM.
